Policy.md

```diff
@@ -26,29 +24,27 @@ The project team shall utilize an existing public-facing website to convey infor
-
-For CMS projects using OSS, each CMS business owner is responsible for assuring that CMS’s use of the open source is according to license.  For a work-for-hire OSS project, the CMS business owner is responsible for selecting an appropriate license. In either case, the TRB is responsible for approval of OSS licenses.
+There is a wide variety of OSS licensing models available, and the CMS project teams are required to research and choose a licensing model that is suitable for their project. Several CMS projects have preferred to dedicate their work in the [Public Domain](https://creativecommons.org/publicdomain/zero/1.0/).  The [Open Source Initiative](https://opensource.org/licenses) maintains a [list of licenses](https://opensource.org/licenses/alphabetical) that the project team may optionally use. The section [Choosing a License](http://wiki.civiccommons.org/Choosing_a_License) at the Civic Commons wiki may also be a useful resource that provides additional help for selection of an appropriate license. The project team should consider the difference between “permissive” and “non-permissive” licenses, warranty/guarantee limitations, attribution, and trademark/IP protections and its impact on the choice of license.
 
-The Open Source Initiative provides a list of [Popular OSI Approved Licenses with strong communities](https://opensource.org/license?categories=popular-strong-community).
+Once an appropriate license is chosen for the project, the full license text should be included in the ‘license’ file of the distribution bundle. Each source code must have a notice at the top to name CMS, or an organization designated explicitly by CMS as the copyright and license holder of the source code and provide a link to the full text of the license.
 
-A useful plain-language resource for Open Source Licenses is [tl;dr legal's verified license resource page](https://www.tldrlegal.com/verified#verified-licenses).
+The project team must perform appropriate due diligence before the publication of their source code, primarily when the project team uses or imports any freely available open source products and libraries to build their software that is released as OSS. The original license of all referenced code and libraries must be included with the distribution of the source, according to the terms of the original license. The CMS Technical Reference Architecture (TRA) Volume IV [“Development and Application Services”](https://www.cms.gov/Research-Statistics-Data-and-Systems/CMS-Information-Technology/CIO-Directives-and-Policies/CIO-IT-Policy-Library-Items/TRA-Volume-IV-Development-and-Application-Services.html?DLPage=3&DLEntries=10&DLSort=0&DLSortDir=ascending) provides additional guidelines for the CMS project team that wish to use the OSS libraries and packaged OSS for their internal consumption or development of new and custom software that is not intended for the publication in the OSS community.
```

Comment:
Just wanted to verify if we wanted to keep all the links in this section. The TRA link is broken.